presto-main/src/main/java/com/facebook/presto/operator/ExchangeClient.java

```diff
@@ -119,6 +122,7 @@ public ExchangeClient(
             int concurrentRequestMultiplier,
             Duration maxErrorDuration,
             boolean acknowledgePages,
+            double averageBytesEMAAlpha,
```

Comment:
`responseSizeExponentialMovingAverageDecayingAlpha`? A little bit verbose, but that's what usually happens when some low level implementation detail being exposed to the higher level interface. After we find a good enough value - the property can be removed.

Reply:
Renamed